rest/route/task.go

```diff
@@ -97,6 +98,111 @@ func (tgh *taskGetHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(taskModel)
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// Handler for the tasks for a project
+//
+//    /projects/{project_id}/versions/tasks
+type projectTaskGetHandler struct {
+	startedAfter   time.Time
+	finishedBefore time.Time
+	projectId      string
+	status         string
```

Comment:
This is contrary to the ticket, but I think (and the discussion doc supports this) that this should be a list of statuses, not a single status